docs/source/Getting-Started/Running-Keystone-on-Hardware.rst

```diff
@@ -62,10 +62,9 @@ or example script below).
 Load Linux Image
 ################
 
-The hifive build process generates a bbl.bin in
-``<build directory>/bbl.bin``. Flash this to the Linux partition on the
-card. (Note that this is a relocated version of the bbl binary used
-for QEMU)
+Flash the ``fw_payload.bin`` generated by the keystone build process above
```

Comment:
@dayeol is this accurate? How is this different from the "Building Keystone" section above?

Reply:
Yes this is accruate. flash `fw_payload.bin` instead of `bbl.bin`